app/src/pages/inside/common/stackTrace/stackTrace.jsx

```diff
@@ -104,6 +104,13 @@ export class StackTrace extends Component {
     this.fetchItems();
   }
 
+  componentDidUpdate(prevProps) {
+    const { items, logItem } = this.props;
+    if (this.isItemsExist() === 0 && prevProps.items !== items && prevProps.logItem !== logItem) {
```

Comment:
Does it necessary check `this.isItemsExist() === 0 && prevProps.items !== items`? Looks like you should check only `logItem` or maybe better `logItem.id`